test/avocado/run-tests

```diff
@@ -231,6 +232,16 @@ def run_avocado(avocado_tests, verbose, browser, download_logs, sit):
         if selenium:
             selenium.kill()
 
+
+def get_test_dict(filters, relative_dir=CURRENT_DIR):
+    test_element = "test"
+    path_element = "path"
+    output = {}
+    for x in Tree(relative_dir).prune(keys=[test_element], filters=filters):
+        output[x.name] = x.data.get(path_element, "") + x.data[test_element]
+    return output
```

Comment:
Adding a new dependency just for this seems rather overkill to me. What does this do that couldn't be done with a simple `glob("selenium-*.py")`?

Reply:
yes, I know that using glob is also fine, but I wanted to introduce new solution for test metadata, tests to be discoverable, be able to filter them easily and run them via external schedulers with complex possibilies how to select tests. eg. be able to tag or mark some tests as tier0, tier1 tests, or be able to filter some tests what are intended for some special envs or need special HW.